.vscode/launch.json

```diff
@@ -11,7 +11,7 @@
 				"serve",
 				"--rebundle"
 			],
-			"cwd": "${workspaceRoot}/../pxt-microbit",
```

Comment:
I don't know that anyone uses this launch command but typically it's better to avoid committing things unrelated to your pr like this as it can distract from the other changes / if anyone else is using it they be surprised. Same with removing the `"name": "checkdocs"`, below~